explorer/interpreter/type_checker.cpp

```diff
@@ -2843,6 +2843,11 @@ auto TypeChecker::TypeCheckExp(Nonnull<Expression*> e,
                 access.set_expression_category(ExpressionCategory::Value);
                 break;
               }
+              case DeclarationKind::AliasDeclaration: {
+                const auto* alias = cast<AliasDeclaration>(member);
+                access.set_expression_category(alias->expression_category());
+                break;
```

Comment:
```suggestion
                return ProgramError(member->source_loc()) << "Member access to aliases is not yet supported.";
```

Reply:
Good suggestion, thanks. I think `access.source_loc()` provides a clearer error message. Refactored to this.